darts/preprocessing/utils.py

```diff
@@ -0,0 +1,139 @@
+"""
+Preprocessing Utils
+-------------------
+"""
+import numpy as np
+
+from typing import TypeVar, Callable, Optional, Tuple, Any, Dict
+
+from darts import TimeSeries
+from darts.logging import raise_if_not, raise_if, get_logger
+from darts.preprocessing.base_transformer import BaseTransformer
+
+logger = get_logger(__name__)
+T = TypeVar('T')
+_callable_with_args_kwargs = Callable[[T, Tuple[Any, ...], Dict[str, any]], T]
+
+
+class _WrapperTransformer(BaseTransformer[T]):
```

Comment:
How would this wrapper be used? What is the typical usecase?

Reply:
I would like this transformer to remains 'private' and be used only by functions `transformer_from_ts_functions` and `transformer_from_values_functions`. So no user would use this class directly. Usecase of this is to be helper wrapper over provided functions. lets say we have only transform and inverse_transform functions and don't want to write transformer for them (like `transform = lambda: x + 1`, `inverse_transfrom = lambda x: x - 1`) then we could just pass this function to `transformer_from_ts_functions` or `transformer_from_values_functions` and have transformer with approppriate functions set.

Reply:
I think this makes sense, being able to pass functions as arguments instead of implementing them in a subclass, kind of like we already do with validators, right?